packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -152,6 +156,12 @@ export const SelectTree = function SelectTree( {
 		value: inputValue,
 	};
 
+	const handleOnClear = () => {
```

Comment:
I would recommend to use always the pattern `handleClear` or `handleClick` for the event listener function.

Reply:
Done in [2a12d5a](https://github.com/woocommerce/woocommerce/pull/49036/commits/2a12d5a9f2cbb25345bb2a44511e850eb1f11f2d)